MiniAn/minian_run.py

```diff
@@ -1,26 +1,17 @@
-
```

Comment:
Maxime, to be able to merge, you need 2 approvals. Please add all 3 of us to review your code

Reply:
oooo I was wondering why antoine was not there.
I'm adding it now! (I thought it was automatic or something)